community/sustainability_effort.md

```diff
@@ -0,0 +1,58 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) to the project and in return have their PRs prioritized for review.
+
+## Where is help needed?
+
+Help is needed for:
+* [reviewing PRs](#reviewing-PRs)
+* [triaging](../docs/CONTRIBUTING.md#triaging-bugs) new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels
+* responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
+* responding to questions in [CNCF Slack](https://argoproj.github.io/community/join-slack) in the `#argo-workflows` and `#argo-wf-contributors` channels 
+
+## Commitment
+
+Argo Workflows has roles as described in the [Contributing](../docs/CONTRIBUTING.md#roles) document.
+Any code contributor in a designated role or with an open Membership request can sign up to participate in this Incentive System.
+
+Participants are expected to try to advance in roles.
+There are different expectations depending on the role:
+- Member and non-member: should average a minimum of 6 hours per week of either [authoring PRs](../docs/CONTRIBUTING.md#authoring-prs) or performing any activities listed above in the ["Where is help needed?"](#where-is-help-needed) section
+  - Note that more time is required in this role in order to learn the codebase and processes to make sufficient progress
+- Reviewer and above: should average a minimum of 2 hours per week performing any activities listed above in ["Where is help needed?"](#where-is-help-needed) section 
+  - Highest priority is to ensure that all PRs labeled ["prioritized-review"](#authoring-prs) have an Assignee
+
+Note that the hours per week listed above is an average over time; it's fine to have weeks of no activity so long as the average meets expectations.
+
+Participants should join the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.
+
+### Reviewing PRs
+
+First priority is to review any PRs which have the `prioritized-review` label, meaning they were authored by a participant of this system (verified by GitHub ID).
+Each of these should have an Assignee: a Reviewer, Approver, or Lead who "owns" reviewing the PR.<br />
+These PRs should be given an initial review within a week.
+The Assignee should then respond to each question or requested modification from the author within a week.
+
+Next, look for PRs with no Assignee. 
+
+For general guidance regarding PR Reviews, see [Reviewing PRs](../CONTRIBUTING.md#reviewing-prs).
```

Comment:
```suggestion
For general guidance regarding PR Reviews, see [the Contributing docs](../CONTRIBUTING.md#reviewing-prs).
```

this section itself is named "Reviewing PRs" (in both docs) so that sounds like a self-reference without seeing that the link is to a different docs page. we can distinguish it by saying "Contributing docs" instead